lua/lint/linters/sqruff.lua

```diff
@@ -0,0 +1,43 @@
+local severities = {
+  Error = vim.diagnostic.severity.ERROR,
+  Warning = vim.diagnostic.severity.WARN,
+}
+
+return {
+  cmd = "sqruff",
+  args = {
+    "lint",
+    "--format=json",
+    "-",
+  },
+  ignore_exitcode = true,
+  parser = function(output, _)
+    if vim.trim(output) == "" or output == nil then
+      return {}
+    end
+
+    if not vim.startswith(output, "{") then
+      vim.notify(output)
+      return {}
+    end
```

Comment:
```suggestion
```

If the parser function raises an error the default error handling reports the error via diagnostics, so this isn't necessary.

Reply:
Sure! I did copy paste this from phpcs if you want to do some pruning :-)